src/renderer/wg_engine/meson.build

```diff
@@ -19,7 +19,10 @@ source_file = [
     'tvgWgShaderTypes.cpp'
 ]
 
-wgpu_dep = dependency('wgpu_native')
+wgpu_dep = []
```

Comment:
@SergeyLebedkin please do not touch wasm related  in this change.

Reply:
> @SergeyLebedkin please do not touch wasm related in this change.

maybe I dont undertand what do ypu mean

Reply:
@hermet  So, I can remove this change from this PR